r/R/install-arrow.R

```diff
@@ -98,7 +98,9 @@ OR_SEE_DEV_GUIDE <- paste0(
 SEE_ARROW_INSTALL <- paste(
   "See the Apache Arrow project installation page",
   "<https://arrow.apache.org/install/>",
-  "for how to install the C++ package from a PPA."
+  "to find pre-compiled binary packages for some common Linux distributions,",
+  "including Debian, Ubuntu, and CentOS. You'll need to install 'libarrow-dev'",
+  "and 'libparquet-dev'."
```

Comment:
`libarrow-dev` and `libparquet-dev` are right package names for Debian and Ubuntu.
But `arrow-devel` and `parquet-devel` are right package names for CentOS.

BTW, we can install only `libparquet-dev` or `parquet-devel` because `libarrow-dev` and `arrow-devel` are automatically installed as dependency.

Reply:
Ok, revised accordingly.